app/helpers/reactjs_helper.rb

```diff
@@ -39,30 +54,21 @@ def select_requested_plugins(plugin_names)
 
   def js_tags_for(requested_plugins)
     requested_plugins.map do |plugin|
-      javascript_include_tag(*webpack_asset_paths(plugin.to_s, :extension => 'js'))
+      javascript_tag("window.tfm.tools.loadPluginModule('/webpack/#{plugin.to_s.tr('-', '_')}','#{plugin.to_s.tr('-', '_')}','./index');".html_safe)
```

Comment:
I had a hard time reading this. How about:
```suggestion
      name = plugin.to_s.tr('-', '_')
      javascript_tag("window.tfm.tools.loadPluginModule('/webpack/#{name}', '#{name}', './index');".html_safe)
```